dll/win32/shell32/lang/pl-PL.rc

```diff
@@ -251,34 +251,34 @@ BEGIN
     PUSHBUTTON "Zaawa&nsowane...", 14028, 158, 170, 70, 14, WS_CHILD | WS_VISIBLE | WS_TABSTOP
 END
 
-IDD_FILE_PROPERTIES DIALOGEX 0, 0, 240, 225 //FIXME: remove that ancient workaround and shrink that dialogs height. Best is to sync it back to the dimensions used in en-US. Otherwise it doesn't fit small monitor resolutions. Check all the dialogs controls then!
+IDD_FILE_PROPERTIES DIALOGEX 0, 0, 240, 205
 STYLE DS_SHELLFONT | WS_CHILD | WS_CAPTION
 CAPTION "Ogólne"
 FONT 8, "MS Shell Dlg", 0, 0, 0x0
 BEGIN
     ICON "", 14000, 16, 7, 32, 32, WS_VISIBLE
     EDITTEXT 14001, 58, 9, 170, 14, ES_LEFT | WS_TABSTOP
     LTEXT "", -1, 8, 29, 221, 1, SS_ETCHEDHORZ | WS_CHILD | WS_VISIBLE
-    LTEXT "Typ pliku:", 14004, 8, 35, 50, 10
+    LTEXT "Typ:", 14004, 8, 35, 50, 10
```

Comment:
```suggestion
    LTEXT "Typ:", 14004, 8, 35, 42, 10
```
width 42 (the answer to all questions) will work here as well, as proven by the other dlg.